app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+app.use(express.json())
+
+const {getTopics} = require("./controllers/controller");
+
+app.get("/api/topics", getTopics);
+
+app.all((err, req, res, next) => {
+    res.status(404).send("Route not found!")
+})
+
+app.use((err, req, res, next) => {
```

Comment:
you haven't added any tests that require this logic yet, please make sure you are using the tests to inform he code you need to write and not have some express boiler plate :0 👍